core/src/main/java/de/danoeh/antennapod/core/service/download/DownloadStatus.java

```diff
@@ -41,62 +41,49 @@
 	 * FEEDFILETYPE_FEEDIMAGE or FEEDFILETYPE_FEEDMEDIA
 	 */
     private final int feedfileType;
+    private final boolean initiatedByUser;
```

Comment:
Instead of adding the field to DownloadStatus, please use `status.getRequest().isInitiatedByUser()`. Then we do not need another copy of the variable.

Reply:
There is no `getRequest` method that I can see. I can add it but not all status' are created with a request, and even so that seems like overkill. Or were you just talking about the constructor that takes a request? I'll clean that up either way.

Reply:
Oh you are right. Then it's OK this way.